src/statsig/index.ts

```diff
@@ -56,3 +60,26 @@ export function getDynamicConfigParams<T extends Record<string, StatsigParameter
     return defaultValues
   }
 }
+
+/* Updates the current Statsig user. If no argument is given, a default StatsigUser
```

Comment:
```suggestion
/**
 * Updates the current Statsig user. If no argument is given, a default StatsigUser
```

to make this a jsdoc